docs/src/content/docs/tr/guides/components.mdx

```diff
@@ -84,7 +84,7 @@ import { Card, CardGrid } from '@astrojs/starlight/components';
 `<Card>` bileşenini kullanarak Starlight'ın stiline uyan kutu içerisinde yer alan içeriği gösterebilirsiniz.
 Birden fazla kartı `<CardGrid>` bileşeni içinde, yeteri kadar boşluk olduğu zaman kartları yan yana sıralı olarak göstermek için sarmalayın.
 
-`<Card>` bileşeni `title` değerine ihtiyaç duyar ve opsiyonel olarak [Starlight'ın kurulu ikonlarından birinin](#tüm-i̇konlar) ismini taşıyan `icon` özelliğini içerebilir.
+`<Card>` bileşeni `title` değerine ihtiyaç duyar ve opsiyonel olarak [Starlight'ın kurulu ikonlarından birinin](#all-icons) ismini taşıyan `icon` özelliğini içerebilir.
```

Comment:
This shouldn’t be changed I don’t think: the heading on the page is “Tüm İkonlar” so it needs the `#` link to match:

```suggestion
`<Card>` bileşeni `title` değerine ihtiyaç duyar ve opsiyonel olarak [Starlight'ın kurulu ikonlarından birinin](#tüm-i̇konlar) ismini taşıyan `icon` özelliğini içerebilir.
```